tests/fake_backends/test_fake_garnet.py

```diff
@@ -0,0 +1,71 @@
+# Copyright 2022-2023 Qiskit on IQM developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Testing fake Garnet backend.
+"""
+from qiskit_aer.noise.noise_model import NoiseModel
+
+from iqm.qiskit_iqm.fake_backends.fake_garnet import IQMFakeGarnet
+from iqm.qiskit_iqm.iqm_backend import IQMTarget
+
+
+def test_iqm_fake_garnet():
+    backend = IQMFakeGarnet()
+    assert backend.num_qubits == 20
+    assert backend.name == 'IQMFakeGarnetBackend'
+
+
+def test_iqm_fake_garnet_connectivity():
```

Comment:
Looks like `pylint` didn't like the copy-pasted code. In the future we want to change things so that this kind of copy-paste is not needed for fake backends, but for now you can just disable the pylint check:
```suggestion
def test_iqm_fake_garnet_connectivity():
    # pylint: disable=duplicate-code
```
This should make the CI pipeline pass, and then I will approve.